setup-jest.mjs

```diff
@@ -2,12 +2,18 @@ import 'zone.js/fesm2015/zone-testing-bundle.min.js';
 import { getTestBed } from '@angular/core/testing';
 import { BrowserDynamicTestingModule, platformBrowserDynamicTesting } from '@angular/platform-browser-dynamic/testing';
 
+let teardown = globalThis.ngJest?.teardown;
 const configuredDestroyAfterEach = globalThis.ngJest?.destroyAfterEach;
-if (configuredDestroyAfterEach !== undefined) {
+
+if (configuredDestroyAfterEach) {
```

Comment:
I think we can add a `console.warn` to tell users to migrate to `teardown` option. Also inform users that `destroyAfterEach` is deprecated.

Reply:
Done. My english is not perfect, so the wording might be a bit akward. Feel free to correct if needed